explainaboard/metrics/f1_score.py

```diff
@@ -2,7 +2,7 @@
 
 from dataclasses import dataclass
 import itertools
-from typing import cast, Optional
+from typing import cast, Optional, Tuple
```

Comment:
Is `Tuple` necessary?

Reply:
I think yes. I previously used `tuple` while it resulted in an error w.r.t following code:
 ```
            gold_spans, pred_spans = cast(
                Tuple[set, set], gen_text_blocks(tags, pred_tags)
            )
```